### DISCUSSION TITLE
Wrong school already registered under a "trust" email address.

### DISCUSSION THREAD

**Discussion by MorganDilling** [community/community#35651] [GitHub Education] [answered]
Hi there. I recently attempted to apply for GitHub student and ran into a little problem.

As the email I use is part of a "trust" (A group of different colleges that use the same email domain), it seems that people have already registered for GitHub student under a different school's name within the trust meaning the proof of enrolment is invalid as it doesn't have that school name in it and there is no way I can change the school name. 

<img width="789" alt="image" src="https://user-images.githubusercontent.com/67484499/194827785-916cf772-5e33-4dd2-893c-803a3693fdf4.png">

I believe this would be considered a bug as there is simply no way around this and I obviously do not have an enrolment letter in the name of the school that is already registered under my email domain.

Any help would be greatly appreciated.

**Comment by MorganDilling** [comment, marked as answer]
This has now been fixed, for me at least. I believe they reset the school for that email.

**Reply by TIMMLOPK** [reply]
I met same problem with you
